sunpy/net/vso/vso.py

```diff
@@ -419,6 +419,14 @@ def fetch(self, query_response, path=None, methods=None, site=None,
 
     def make_getdatarequest(self, response, methods=None, info=None):
         """ Make datarequest with methods from response. """
+        
+        #    Pass back the Apache session ID to the VSO if it exists in the response
+
+        for item in response:
+           for k in item.keys():
+              if k == 'Info Required':
+                 info['required'] = item['Info Required']
```

Comment:
The indentation is wrong in this section, indentation should always be 4 spaces.